app/components/AllProblemsBoard.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import {Table} from 'react-bootstrap'
+import {ProblemSetData} from 'data/Data'
+
+export default class AllProblemsBoard extends React.Component {
+
+  constructor (props) {
+    super(props)
+    this.state = ProblemSetData
+  }
+
+  render () {
+    return (
+      <div>
+        <h2 className='page-title'> Full Problem List </h2> 
+        <Table striped bordered condensed hover>
+          <thead>
+            <th>   Week #  </th> 
```

Comment:
Why are these titles padded with spaces? Do you want them to be centered? If so you they it's done is with CSS, AFAIK.

Reply:
I do not want them to be centered in the page, but just a code alignment.